game/static/game/css/avatar.css

```
@@ -0,0 +1,331 @@
+    flex-direction: column;
+    gap: 16px;
+}
+
+/* Hidden real file input */
+.file-input {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    opacity: 0;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)